### PR TITLE
runtime: Fix lint issues

### PR DESCRIPTION
**Pull request by fidencio**
Apart from the work done on the tests repo to ignore the `govet` issues, let's fix the few others that are easily fixed as part of this PR.

 Fixes: #2204

### CODE REVIEW

**Review by amshinde — APPROVED** [review]

**Review by devimc — APPROVED** [review]
thanks @fidencio

File: src/runtime/virtcontainers/pkg/agent/protocols/client/client.go [hunk 1]
```diff
@@ -428,9 +428,7 @@ func HybridVSockDialer(sock string, timeout time.Duration) (net.Conn, error) {
 
 // just for tests use.
 func MockHybridVSockDialer(sock string, timeout time.Duration) (net.Conn, error) {
-	if strings.HasPrefix(sock, "mock:") {
-		sock = strings.TrimPrefix(sock, "mock:")
```
Comment (Jakob-Naucke) [inline src/runtime/virtcontainers/pkg/agent/protocols/client/client.go:432]: smart linter
LGTM, thanks @fidencio

**Review by Jakob-Naucke — APPROVED** [review]

**Pull request merged**